src/util/methodDataUtil.js

```diff
@@ -132,6 +132,16 @@ function getFunctionMaxArity(version, name) {
     return getMethodData(version).args[name] || Infinity
 }
 
+/**
+ * Returns whether or not a method is supported in the given version of lodash
+ * @param {number} version
+ * @param {string} name
+ * @returns {boolean}
+ */
+function isMethodSupported(version, name) {
+    return getMethodData(version).args.hasOwnProperty(name)
```

Comment:
I don't think we can rely on args as wrapper and alias methods won't be covered.

An example is trying to test for `str.replace` in v4.

```
const methodData = getMethodData(version)

return _(methodData.aliases).keys().concat(methodData.wrapper).concat(methodData.args).includes(method)
```


Or maybe it's a bug that `args` is incomplete.